migration/migration.go

```diff
@@ -75,15 +75,16 @@ func createCommon(typeName string, ctx context.Context, witr models.WorkItemType
 	case models.NotFoundError:
 		stString := "string"
 		_, err := witr.Create(ctx, nil, typeName, map[string]app.FieldDefinition{
-			"system.title":       app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: true},
-			"system.description": app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: false},
-			"system.creator":     app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: true},
-			"system.assignee":    app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: false},
+			"system.title":          app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: true},
+			"system.description":    app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: false},
+			"system.creator":        app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: true},
+			"system.assignee":       app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: false},
+			"system.remote_item_id": app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: false},
 			"system.state": app.FieldDefinition{
 				Type: &app.FieldType{
 					BaseType: &stString,
 					Kind:     "enum",
-					Values:   []interface{}{"new", "in progress", "resolved", "closed"},
+					Values:   []interface{}{"new", "open", "in progress", "resolved", "closed"},
```

Comment:
@sbose78 We should not introduce new states into system rather we need to map it. I have added a ticket for the same here: https://github.com/almighty/almighty-core/issues/272 

I think this is not critical for the sprint no. 120 , so we can merge this change.